### ISSUE TITLE
Nheko crashes on Mobian when attempting login to SSO server

### ISSUE THREAD

**Issue by mefuller** [Nheko-Reborn/nheko#808]
### Describe the bug

Entering a matrix ID associated with Element One (one.ems.host, fedora.im), which utilizes SSO, results in Nheko immediately crashing once the ID is entered.

### To Reproduce

1. Open Nheko
2. Attempt to login to existing account
3. enter ID:one.ems.host
4. Click on device name field
5. Crash

### What happened?

Nheko crashed before and without providing an opportunity to login

### Expected behavior

Nheko should either direct to SSO webpage or at least state that SSO isn't supported

### Version

0.8.0+really0.7.2-4

### Operating system

Linux

### Installation method

Some repository (AUR, homebrew, distribution repository, PPA, etc)

### Desktop Environment

Mobian/Phosh

### Did you use profiles?

- [ ] Profiles used?

### Relevant log output

```shell
<empty>
```

### Backtrace

```shell
unfortunately presently beyond my means on PinePhone
```

**Comment by sc0w** [comment]
it was already fixed, you can install nheko 0.8.2 (fixed) from debian sid

**Comment by deepbluev7** [comment]
Yep, this should be fixed in anything newer than 0.7.2, which you are using.
